plugin/pkg/admission/resourcequota/controller.go

```diff
@@ -549,7 +550,7 @@ func (e *quotaEvaluator) Evaluate(a admission.Attributes) error {
 	select {
 	case <-waiter.finished:
 	case <-time.After(10 * time.Second):
-		return fmt.Errorf("timeout")
+		return apierrors.NewInternalError(fmt.Errorf("resource quota evaluates timeout"))
```

Comment:
I don't understand this error message. Is it trying to say `resource quota evaluation took too long`?

Reply:
yes

Reply:
I like "resource quota evaluation took too long" better.